scratchpad/components/todo/index.ts

```diff
@@ -0,0 +1,10 @@
+import {TodoController} from './controllers/TodoController.ts';
+import {WebController} from './controllers/WebController.ts';
+import {TodoRepo} from './bindings/TodoRepo.ts';
+import {TodoRepo} from './bindings/AuthStrategy.ts';
+import {Role} from './bindings/Role.ts';
+
+export class Todo {
+  controllers = [TodoController, WebController];
+  bindings = [TodoRepo, AuthStrategy, Role];
+}
```

Comment:
Each component contributes extensions to the parent container. The code above allows the component to declare what extensions should be added the container. Can we also support another style, such as:

```
class TodoComponent {
  constructor(@inject('context') ctx: Context) {
  }
  start() {
    ctx.bind(...);
  }
  stop() {
    ctx.unbind(...);
  }
}
```

We also need to support lifecycle of components.